visualization/include/pcl/visualization/common/actor_map.h

```diff
@@ -69,7 +69,8 @@ namespace pcl
       public:
         CloudActor () : color_handler_index_ (0), geometry_handler_index_ (0) {}
 
-        virtual ~CloudActor ()
+        // Workaround for LLVM clang-tidy issue https://github.com/llvm/llvm-project/issues/56447
+        virtual ~CloudActor () // NOLINT
```

Comment:
The default destructor would call the destructor of every member, which would do more or less the same as `clear()` for `std::vector`, right? Or am I missing something?

Reply:
You're correct, I was distracted by the fact that the destructor had a body.